js/ui/drop_down_editor/ui.drop_down_editor.js

```diff
@@ -625,6 +625,13 @@ const DropDownEditor = TextBox.inherit({
     },
 
     _dimensionChanged: function() {
+        // This uses for hiding of popup on window resize if editor was hidden (T1133813)
```

Comment:
1. Add "NOTE" prefix
2. "resize observer", not "observable"!
3. Please, check the text with redactors